app/components/satis/dropdown/component.html.slim

```diff
@@ -12,7 +12,7 @@ div.satis-dropdown data-action="keydown->satis-dropdown#dispatch" data-controlle
         .h-12.p-1.flex.rounded
           .flex.flex-auto.flex-wrap
             / Input where you can search
-            input.p-1.px-2.appearance-none.outline-none.w-full.sts-dropdown-input.text-gray-800.dark:text-gray-300 data-action="input->satis-dropdown#search" data-satis-dropdown-target="searchInput" placeholder=placeholder autofocus=options[:autofocus]
+            input.text.p-1.px-2.appearance-none.outline-none.w-full.sts-dropdown-input.text-gray-800.dark:text-gray-300 data-action="input->satis-dropdown#search" data-satis-dropdown-target="searchInput" placeholder=placeholder autofocus=options[:autofocus] ondblclick="this.select()"
```

Comment:
This needs to work with stimulus events (dblclick) and then trigger the select of the input. Don't put normal JS directly in html.

https://onrails.blog/2018/04/11/lets-make-an-editable-header-using-stimulus-js/